elasticsearch/install.sh

```diff
@@ -2,6 +2,7 @@
 
 set -ex
 
+yum -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
```

Comment:
I think that is a holdover for when we had curator as part of the elasticsearch image.  I will remove that.